src/core/tests/eval.cpp

```diff
@@ -2655,6 +2656,1216 @@ TEST(eval, evaluate_fake_quantize_dynamic_input) {
+}
+
+TEST(eval, evaluate_fake_convert_f16_matching_f8_to_f8e4m3_scale_false) {
+    using namespace testing;
+    constexpr auto et = element::f16;
+
+    std::vector<float16> input_data{
+        0.f,       0.001953125f, 0.00390625f, 0.005859375f, 0.0078125f, 0.009765625f, 0.01171875f, 0.013671875f,
+        0.015625f, 0.017578125f, 0.01953125f, 0.021484375f, 0.0234375f, 0.025390625f, 0.02734375f, 0.029296875f,
+        0.03125f,  0.03515625f,  0.0390625f,  0.04296875f,  0.046875f,  0.05078125f,  0.0546875f,  0.05859375f,
+        0.0625f,   0.0703125f,   0.078125f,   0.0859375f,   0.09375f,   0.1015625f,   0.109375f,   0.1171875f,
+        0.125f,    0.140625f,    0.15625f,    0.171875f,    0.1875f,    0.203125f,    0.21875f,    0.234375f,
+        0.25f,     0.28125f,     0.3125f,     0.34375f,     0.375f,     0.40625f,     0.4375f,     0.46875f,
+        0.5f,      0.5625f,      0.625f,      0.6875f,      0.75f,      0.8125f,      0.875f,      0.9375f,
+        1.f,       1.125f,       1.25f,       1.375f,       1.5f,       1.625f,       1.75f,       1.875f,
+        2.f,       2.25f,        2.5f,        2.75f,        3.f,        3.25f,        3.5f,        3.75f,
+        4.f,       4.5f,         5.f,         5.5f,         6.f,        6.5f,         7.f,         7.5f,
+        8.f,       9.f,          10.f,        11.f,         12.f,       13.f,         14.f,        15.f,
+        16.f,      18.f,         20.f,        22.f,         24.f,       26.f,         28.f,        30.f,
+        32.f,      36.f,         40.f,        44.f,         48.f,       52.f,         56.f,        60.f,
+        64.f,      72.f,         80.f,        88.f,         96.f,       104.f,        112.f,       120.f,
+        128.f,     144.f,        160.f,       176.f,        192.f,      208.f,        224.f,       240.f,
+        256.f,     288.f,        320.f,       352.f,        384.f,      416.f,        448.f};
+
+    const auto data_shape = Shape{input_data.size()};
+
+    auto data = make_shared<ov::op::v0::Parameter>(et, data_shape);
+    auto scale = op::v0::Constant::create(et, Shape{1}, {1.f});
+    auto shift = op::v0::Constant::create(et, Shape{1}, {0.f});
+
+    auto op = make_shared<op::v13::FakeConvert>(data, scale, shift, "f8e4m3");
+    auto model = make_shared<Model>(OutputVector{op}, ParameterVector{data});
+
+    auto result = ov::Tensor();
+    auto out_vector = ov::TensorVector{result};
+    auto in_vector = ov::TensorVector{make_tensor<et>(data_shape, input_data)};
+    ASSERT_TRUE(model->evaluate(out_vector, in_vector));
+    result = out_vector.at(0);
+
+    EXPECT_EQ(result.get_element_type(), et);
+    EXPECT_EQ(result.get_shape(), data_shape);
+    EXPECT_THAT(read_vector<float16>(result), Pointwise(FloatEq(), input_data));
+}
+
+TEST(eval, evaluate_fake_convert_f16_matching_f8_to_f8e4m3_scale_1) {
+    using namespace testing;
+    constexpr auto et = element::f16;
+
+    std::vector<float16> input_data{
```

Comment:
is this table repeated in other tests if yes make one common?

Reply:
The tests are going to be moved to the "template" tests, as a next step, then refactored if needed.